### ISSUE TITLE
Unexpected behaviour when using type() on triple key combinations

### ISSUE THREAD

**Issue by kensoh** [RaiMan/SikuliX1#160]
Hi Raimund,

A friend encountered a situation where using type() to send ctrl + alt + del was not able to trigger the lock screen on Windows 10. He is able to manually open the lock screen with these keys.

The code used is below and we are using SikuliX v1.1.4 -

```
type(Key.DELETE, KeyModifier.CTRL + KeyModifier.ALT)
```

I can see following logs from SikuliX but for some reason there is no action from Windows, as if the keystrokes combination has not been triggered and received by Windows -

```
[log] ( Ctrl+Alt )  TYPE "#DEL."
```

Another combination that does not work is ctrl + shift + del on Chrome browser on Windows -

```
type(Key.DELETE, KeyModifier.CTRL + KeyModifier.SHIFT)
```

However, there are some combinations that work, for eg cmd + shift + 4 on macOS, will generate below SikuliX log and the macOS opens the screenshot cursor. So the problem doesn't seem to be ability to handle triple key combinations, but maybe for some reason does not work for some edge cases.
```
[log] ( ⌘+⇧ )  TYPE "4"
```

Posting an issue here so other users and you may have encountered something like this and has an idea or workaround to trigger such keystroke combinations. Thanks for reading!

(observation originally spotted at https://github.com/kelaberetiv/TagUI/issues/488)

**Comment by RaiMan** [comment]
> type() to send ctrl + alt + del 

I am rather sure, that this key combination is blocked internally if not issued from a real keyboard.
Hence it results in a NOOP.

> Another combination that does not work is ctrl + shift + del

Just tested on Windows 10 and Chrome:
`type(Key.DELETE, KeyModifier.SHIFT + KeyModifier.CTRL)` and
`type(Key.DELETE, Key.SHIFT + Key.CTRL)`
both work as expected and open the page `delete browser history` from the chrome preferences

my test script: (with Chrome already open)
```
switchApp("chrome")
wait(1)
type(Key.DELETE, KeyModifier.SHIFT + KeyModifier.CTRL)
wait(3)
```

**Comment by kensoh** [comment]
Thanks Raimund for your advice!

I get it now, you are saying that ctrl + alt + del won't trigger the lock screen unless it is issued from the keyboard, I found this old thread of this known issue - https://bugs.launchpad.net/sikuli/+bug/548651

For ctrl + shift + delete I can't seem to get it working on a borrowed Windows 10. It may be related to the known turning off numlock issue (https://bugs.launchpad.net/sikuli/+bug/894312 and https://answers.launchpad.net/sikuli/+question/100368). I accidentally lock myself out on the laptop when trying to find and press numlock key. Will test and update again after owner unlocks it.

**Comment by kensoh** [comment]
Hi Raimund,

I've tested on a Windows 10 laptop. For ctrl + shift + delete, I confirm that it is due to the known numlock issue with Java affecting shift key (https://bugs.launchpad.net/sikuli/+bug/894312 and https://answers.launchpad.net/sikuli/+question/100368). Below are my replication steps to confirm that it is due to the known issue.

Doing the following fails (I manually focus on Chrome browser while SikuliX waits) -
```
wait(5); type(Key.DELETE, KeyModifier.SHIFT + KeyModifier.CTRL)
```

Found out that numlock islockon status is True, and press numlock to turn it off -
```
print(Env.isLockOn(Key.NUM_LOCK))
type(Key.NUM_LOCK)
print(Env.isLockOn(Key.NUM_LOCK))
```

Doing the following works and opens up the Chrome delete browser history page -
```
wait(5); type(Key.DELETE, KeyModifier.SHIFT + KeyModifier.CTRL)
```

I'll close this issue now as both observations are known issues. Thanks Raimund!
